cherrypy/_cpnative_server.py

```diff
@@ -95,7 +95,7 @@ def send_response(self, status, headers, body):
         req = self.req
 
         # Set response status
-        req.status = str(status or '500 Server Error')
+        req.status = status or b'500 Server Error'
```

Comment:
@jaraco there's ` or ''` over the place defaulting to bytes/unicode all over the place. Shall we try `from __future__ import unicode_strings`?

Reply:
I'd very much like to do that. It was a feature that wasn't available in the early implementations (Python 2.3 compatibility and all), and even some people still advise against its use. It's my opinion, however, that using `from __future__ import unicode_literals` is the best way to achieve compatibility. Let's do it as time permits.

Reply:
My concern is that from code it's not always obvious what kind of data the variable is expected to hold. Should we introduce some convention of prefixing `b_` or suffixing `_b` to improve the situation?